### DISCUSSION TITLE
How to set proxy when create session

### DISCUSSION THREAD

**Discussion by pystar1007** [FlareSolverr/FlareSolverr#784] [General]
I am not sure how to set proxy when create session.

I use executable in Windows.

requests.post(flaresolverr_url, json ={"cmd": "sessions.create"})

How to add proxy url here? 

Would you send me example?

thanks

**Comment by ngosang** [comment]
```
curl --request POST \
  --url http://127.0.0.1:8191/v1 \
  --header 'Content-Type: application/json' \
  --data '{
	"cmd": "sessions.create",
	"proxy": {
		"url": "http://127.0.0.1:8888"
	}
}'
```
